app/src/script.js

```diff
@@ -105,39 +118,56 @@ async function updateConnectedAccount(state, { account }) {
 
 async function newTokenRequest(
   state,
-  { requestId, requesterAddress, depositToken, depositAmount, requestAmount, date },
+  { requestId, requesterAddress, depositToken, depositAmount, requestAmount },
   settings,
   blockNumber
 ) {
-  const { account, requests } = state
-  let status
-  if (!account) return state
+  try {
+    const { account, requests } = state
+    let status
+    if (!account) return state
```

Comment:
I think the Org requests should be displayed regardless if there's a signed account or not?